apps/jenkins/src/main/fabric8/openshift-deployment.yml

```diff
@@ -76,6 +76,8 @@ spec:
             secretKeyRef:
               name: jenkins-recommender-api-token
               key: token
+        - name: JENKINS_ROOT_URL
+          value: "https://jenkins.openshift.io"
```

Comment:
We need this value to be different for prod-preview (https://jenkins.prod-preview.openshift.io) and production  (https://jenkins.openshift.io) - how will we achieve this?

Reply:
How about `Value: "${URL_PLACEHOLDER}"` and CI/CD or deployment tools can substitute this value for target environment.

@vpavlin @rupalibehera @aslakknutsen